### PR TITLE
XW-1209 | Load Internal tracking before Ember starts

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1209

## Description

Load Internal tracking before Ember starts.

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/auth/views/_layouts/auth.hbs [hunk 1]
```diff
@@ -69,6 +69,8 @@
 			M.prop('isGASpecialWiki', true);
 			M.prop('initialPageView', true, true);
 			M.prop('urlTitleParam', '');
+			M.prop('pageTitle', '{{i18n title}} - Wikia');
```
Comment (rogatty) [inline front/auth/views/_layouts/auth.hbs]: This is duplicated in this file. Can we move setting it to JS and pass as a context to this template?

File: front/main/app/index.html [hunk 2]
```diff
@@ -91,6 +91,7 @@
 	M.loadDOMResource('/front/main/assets/common.svg');
 </script>
 
+{{> tracking-internal}}
```
Comment (rogatty) [inline front/main/app/index.html]: Can we move it next to the other tracking partials?

**Comment by rogatty** [comment]
One minor thing and it'll LGTM

File: front/common/baseline/tracker-internal.js [hunk 3]
```diff
@@ -117,36 +109,46 @@ export default class Internal {
 	 * @param {Element} [script=document.createElement('script')]
 	 * @returns {void}
 	 */
-	loadTrackingScript(url, script = document.createElement('script')) {
+	function loadTrackingScript(url, script = document.createElement('script')) {
```
Comment (hakubo) [inline front/common/baseline/tracker-internal.js]: can we use $script here as well?

**Comment by hakubo** [comment]
one major thing and it'll lgtm :)

**Comment by hakubo** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Pull request merged**